tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -1189,8 +1189,9 @@ TEST_F(Caffe2ImporterTest, importResizeNearest) {
   ASSERT_TRUE(resizeNearestNode);
   // We have one input and one output.
   EXPECT_EQ(mod.getPlaceholders().size(), 2);
-  auto heightScale = resizeNearestNode->getHeightScale();
-  auto widthScale = resizeNearestNode->getWidthScale();
+  auto scale = resizeNearestNode->getScale();
+  auto heightScale = scale[1];
+  auto widthScale = scale[2];
```

Comment:
nit: Probably a good idea to check `scale[0]` and `scale[3]` are `1`.

Reply:
This is fixed

Reply:
Done